src/main/java/com/online/cinema/util/HibernateUtil.java

```diff
@@ -0,0 +1,28 @@
+package com.online.cinema.util;
+
+import com.online.cinema.exceptions.DataProcessingException;
+import com.online.cinema.model.Movie;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+        
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            Configuration configuration = new Configuration();
+            configuration.addAnnotatedClass(Movie.class);
```

Comment:
let's better use hibernate.cfg.xml
so that we added new entities there and not here

it will be better from the point of distributing logic